controllers/user.js

```diff
@@ -347,11 +347,51 @@ exports.login_user = (req, res, next) => {
                 middle_initial: result[0].middle_initial,
                 last_name: result[0].last_name
             };
-            
+
             logger.logg(req.session.user.teacher_id, req.session.user.first_name + ' ' + req.session.user.middle_initial + ' ' + req.session.user.last_name + ' logged in to StudEx.');
             res.item({code :'USER200', message:'User succesfully logged in.'}).send();
         }
     }
 
     start();
 };
+
+exports.change_password = (req, res, next) => {
+
+    const data = util.get_data(
+      {
+            password: ''
```

Comment:
Require the old password. Check if the old password matches the one in the database